hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -108,8 +110,10 @@ SqlCreate SqlCreateType(Span span, boolean replace) :
     ]
     name = CompoundIdentifier()
     columns = TypeColumns()
-    <OPTIONS>
-    sqlOptions = SqlOptions()
+    [
```

Comment:
I made it optional because it's optional in every other clause. The fact that we require some options on semantic level doesn't mean they should be required on syntactic level. For example, if you use imap type in mapping, you must specify some options. But for some other connector type you don't have to. 

But for now it's the same so I don't care...